src/components/Filters/components/ConnectedFilterControl/ConnectedFilterControl.scss

```diff
@@ -78,6 +78,19 @@ $stacking-order: (
   }
 }
 
+.RightContainer.queryFieldHidden {
+  .Item:first-of-type > * > * {
```

Comment:
Can we be more specific to what we're targeting here?

Reply:
I updated this to `.Item > div > button` since that's the structure of all of our items.

I also updated all of the existing `.Item > * > *` selectors to `.Item > div > button` as well so that if the DOM structure changes, all of these selectors break. Hopefully this makes it easier to avoid any UI regressions.